viewer/css/app.css

```diff
@@ -56,6 +56,27 @@ body {
   background-color: #fff;
 }
 
+.embed {
+  color: #6a6a6a;
+}
+
+.embed #icon-list {
+  margin: 0;
+}
+
+.embed #search-input {
+  top: 0;
```

Comment:
This rule looks unused (there's no search-input in the HTML page), can you remove it ?